plugins/Denizen/scripts/restraining_order.dsc

```diff
@@ -0,0 +1,33 @@
+restraining_order:
+    type: item
+    display name: <&f>Restraining Order
+    material: paper
+    flags:
+        uuid: <util.random_uuid>
+    lore:
+        - <&7>Right click to assign to a player!
+
+restraining_order_handler:
+    type: world
+    debug: true
+    events:
+        on player right clicks entity with:restraining_order type:player:
```

Comment:
This can be
`on player right clicks player with:restraining_order`

Reply:
doesnt work like that apparently

Reply:
Huh. It worked when I tested that. I probably messed something up then haha 😄

Reply:
maybe i messed soemthng up

Reply:
Both are valid and functional